extension/src/background/helpers/account.ts

```diff
@@ -32,3 +34,9 @@ export const addAccountName = ({
 
 export const getIsTestnet = () =>
   JSON.parse(localStorage.getItem(IS_TESTNET_ID) || "false");
+
+export const getIsValidatingMemo = () =>
+  JSON.parse(localStorage.getItem(IS_VALIDATING_MEMO_ID) || "true");
+
+export const getIsValidatingSafety = () =>
+  JSON.parse(localStorage.getItem(IS_VALIDATING_SAFETY_ID) || "true");
```

Comment:
dealing with localstorage is getting a bit duplicative, but i'm hoping to possibly replace all of this soon with IndexedDB